lib/Twig/Lexer.php

```diff
@@ -51,22 +51,23 @@ public function __construct(Twig_Environment $env, array $options = array())
         $this->env = $env;
 
         $this->options = array_merge(array(
-            'tag_comment'     => array('{#', '#}'),
-            'tag_block'       => array('{%', '%}'),
-            'tag_variable'    => array('{{', '}}'),
-            'whitespace_trim' => '-',
-            'interpolation'   => array('#{', '}'),
+            'tag_comment'           => array('{#', '#}'),
+            'tag_block'             => array('{%', '%}'),
+            'tag_variable'          => array('{{', '}}'),
+            'whitespace_all_trim'   => '-',
```

Comment:
renaming the key is a BC break for people instantiating the lexer with custom options